package.json

```diff
@@ -54,8 +82,21 @@
       }
     ]
   },
+  "dependencies": {
+    "vscode-languageclient": "8.1.0"
+  },
   "devDependencies": {
-    "@vscode/vsce": "^2.19.0",
-    "vscode-tmgrammar-test": "^0.1.1"
+    "@types/node": "20.3.0",
+    "@types/vscode": "1.79.0",
+    "@typescript-eslint/eslint-plugin": "5.59.9",
+    "@typescript-eslint/parser": "5.59.9",
+    "@vscode/vsce": "2.19.0",
+    "esbuild": "0.18.0",
+    "eslint": "8.42.0",
+    "eslint-config-xo": "0.43.1",
+    "eslint-config-xo-typescript": "0.57.0",
+    "tmlanguage-generator": "0.4.1",
+    "typescript": "5.1.3",
+    "vscode-tmgrammar-test": "0.1.1"
```

Comment:
Let's keep all of these with a caret (`^`) to allow security patches to go through. That is until we have dependabot enabled and fully automated release process.